### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-generic.git`](https://github.com/cisagov/skeleton-generic.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/guacamole-composition.git guacamole-composition
    cd guacamole-composition
    git remote add skeleton https://github.com/cisagov/skeleton-generic.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the branch,
   `commit`, and `push` your changes:

    ```console
    git add .github/CODEOWNERS .gitignore 
    git commit --message="Resolve Lineage conflicts."
    git push --force --set-upstream origin lineage/skeleton
    ```

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
:zap:

**Review by dav3r — APPROVED** [review]
👍

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Looks good, but I would like the `test` job to better mirror the changes that have been made to the `lint` job.
https://github.com/cisagov/guacamole-composition/blob/96c649cef9ab131f29a7eebf31266d3d2d7131ea/.github/workflows/build.yml#L57-L66
should be updated to mirror
https://github.com/cisagov/skeleton-python-library/blob/a5378b01689137ac6c78aef957348e87aa12f875/.github/workflows/build.yml#L57-L66

Also, https://github.com/cisagov/guacamole-composition/blob/96c649cef9ab131f29a7eebf31266d3d2d7131ea/.github/workflows/build.yml#L50 should be `@v2`

**Comment by jsf9k** [comment]
> Looks good, but I would like the `test` job to better mirror the changes that have been made to the `lint` job.

See commit e7b557e.  Great catch!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -47,23 +48,23 @@ jobs:
...
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
         with:
           python-version: 3.8
       - name: Store installed Python version
         run: |
           echo "::set-env name=PY_VERSION::"\
           "$(python -c "import platform;print(platform.python_version())")"
       - name: Cache pip test requirements
```
Comment (mcdonnnj) [inline .github/workflows/build.yml]: I think this should be "Cache testing environments" to mirror the language changes made to the name of the `lint` cache step:
https://github.com/cisagov/guacamole-composition/blob/e7b557eb949d3a42cc194c2efdaa3f3cf06f9788/.github/workflows/build.yml#L26

**Review by mcdonnnj — APPROVED** [review]
Looks good. One last nitpick if you're inclined,

File: .github/workflows/build.yml (hunk 1 above)
Comment (jsf9k) [reply .github/workflows/build.yml]: Changed in commit aa58a73.

Comment (mcdonnnj) [reply .github/workflows/build.yml]: Thanks for taking the time!

**Pull request merged**